src/Client.php

```diff
@@ -150,8 +177,7 @@ function(): void
                     {
                         if($this->connection->connected() === false) {
                             $this->state = self::STATE_NOT_CONNECTED;
-
-                            throw Exception\ClientException::disconnected();
```

Comment:
Throwing an exception into the nirvana of the event loop error handler does not really help